coverage/test/io.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+//DeleteDir deletes a directory on disk
+func DeleteDir(dir string) {
```

Comment:
Again, why is this necessary? It's the same amount of code to do this:

```go
if err := os.RemoveAll(dir); err != nil {
    return fmt.Errorf("failed to remove artifacts directory: %v", err)
}
```

as it does to do this

```go
if err := DeleteDir(dir); err != nil {
    return err
}
```